src/vcf/meta_entry.cpp

```diff
@@ -103,19 +103,26 @@ namespace ebi
             check_sample(value);
         }
     }
+
+    inline void MetaEntryVisitor::check_key_is_present(std::string const & field, std::string const & key, int key_count) const
+    {
+        if (key_count == 0) {
+            throw new MetaSectionError{entry.line, field + " metadata does not contain a field called '" + key + "'"};
```

Comment:
After checking the specification and given that metadata definitions are key-value pairs, I think this string should be written as `field + " metadata does not contain a key called '" + key + "'"`. Even the variable is already called `key`!